### PR TITLE
Add swagger editor for API

### PR DESCRIPTION
**Pull request by dilan-premarathna**
This PR is related to the introduction of an embedded swagger editor for the APIs. The generated swagger file will be stored in the registry and can then be modified with the swagger editor.

Fixes: wso2/product-ei/issues/3328

### CODE REVIEW

File: components/mediation-ui/org.wso2.carbon.rest.api.ui/pom.xml [hunk 1]
```diff
@@ -89,6 +89,20 @@
...
             <artifactId>org.wso2.carbon.logging</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.mediation</groupId>
+            <artifactId>org.wso2.carbon.mediation.transport.handlers</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.commons</groupId>
+            <artifactId>org.wso2.carbon.tenant.common</artifactId>
+            <version>4.6.43</version>
```
Comment (Nirothipan) [inline components/mediation-ui/org.wso2.carbon.rest.api.ui/pom.xml]: Shall we get the version from parent pom

File: components/mediation-ui/org.wso2.carbon.rest.api.ui/src/main/resources/web/api/swaggereditor.jsp [hunk 2]
```diff
@@ -0,0 +1,126 @@
+<%--
+  ~  Copyright (c) 2008, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Nirothipan) [inline components/mediation-ui/org.wso2.carbon.rest.api.ui/src/main/resources/web/api/swaggereditor.jsp]: Please update  year

File: components/mediation-ui/org.wso2.carbon.rest.api.ui/pom.xml (hunk 1 above)
Comment (dilan-premarathna) [reply components/mediation-ui/org.wso2.carbon.rest.api.ui/pom.xml]: Done

File: components/mediation-ui/org.wso2.carbon.rest.api.ui/src/main/resources/web/api/swaggereditor.jsp (hunk 2 above)
Comment (dilan-premarathna) [reply components/mediation-ui/org.wso2.carbon.rest.api.ui/src/main/resources/web/api/swaggereditor.jsp]: Done

**Comment by sdkottegoda** [comment]
Closing this PR since the funcationality is incomplete. Please re-open once completed.

**Pull request closed without merging**